app/workers/check_worker.rb

```diff
@@ -1,6 +1,21 @@
 class CheckWorker
   include Sidekiq::Worker
 
+  sidekiq_options retry: 10
```

Comment:
Do you know how long 10 retries takes until? I know 25 ~= 21 days so not sure if this is measured in days?